### ISSUE TITLE
find a differ supporting object '250' of type 'string'. NgFor only supports binding to Iterables such as Arrays

### ISSUE THREAD

**Issue by hevan** [swimlane/ngx-datatable#950]
**I'm submitting a ...**  (check one with "x")
```
[ ] bug report => search github for a similar issue or PR before submitting
[ ] feature request
[ ] support request => Please do not submit support request here, post on Stackoverflow or Gitter
```

**Current behavior**

**Expected behavior**

**Reproduction of the problem**

**What is the motivation / use case for changing the behavior?**

**Please tell us about your environment:**

Table version: 10.2.0

Angular version: 4.3.3

find a differ supporting object '250' of type 'string'. NgFor only supports binding to Iterables such as Arrays

* **Browser:** [all | Chrome XX | Firefox XX | IE XX | Safari XX | Mobile Chrome XX | Android X.X Web Browser | iOS XX Safari | iOS XX UIWebView | iOS XX WKWebView ]

* **Language:** [all | TypeScript 2.4.2 | ES6/7 | ES5]

**Comment by ceoaliongroo** [comment]
+1 Same error after update to version 10.2.1

**Comment by wizarrc** [comment]
Where is this happening? Can you provide a repro?

**Comment by ceoaliongroo** [comment]
Sorry i can't reproduce the error at the moment.

**Comment by hevan** [comment]
when rows data have  fieldname  "value" and all this Field value is string number; like "value":"250"

**Comment by wizarrc** [comment]
@hevan I'm not sure I'm following. So your row data is something like this:?
`const rows = [{ value: "250", }, { value: "251" }]`

What are you using for your columns?

Do you have a plunker you can link to?

**Comment by hevan** [comment]
@wizarrc like that。but I haved to change the field name and change string to number, all ok! Thank you!!!

**Comment by ceoaliongroo** [comment]
So it's a bug, or theres is some datatype definition to the rows. Right?

**Comment by shriramMani** [comment]
Seems like. I had the same issue when my data looks like this
[
    {
      "key": "Ethel Price",
      "value": "Johnson"
    },
    {
      "key": "Claudine Neal",
      "value": "Sealoud"
    },
    {
      "key": "Beryl Rice",
      "value": "Velity"
    }
]

But reading thru the comments here, and after changing it to 

[
    {
      "key": "Ethel Price",
      "propValue": "Johnson"
    },
    {
      "key": "Claudine Neal",
      "propValue": "Sealoud"
    },
    {
      "key": "Beryl Rice",
      "propValue": "Velity"
    }
]

it worked fine. Thank you

**Comment by ceoaliongroo** [comment]
@shriramMani you answer clarify a lot the comments

thanks!

**Comment by maxgomes92** [comment]
Problem solved on release 10.3.0
